public/js/user-actions.js

```diff
@@ -27,6 +27,7 @@ module.exports = assign({}, {
       console.log('login succeeded');
       this.setCurrentUser({
         email: data.buyer_email,
+        payment_methods: data.payment_methods,
```

Comment:
yeah, we're thinking of replacing it with fetch, just haven't got around to it yet. https://github.com/mozilla/payments-ui/issues/84